### DISCUSSION TITLE
Help hls bandwidth limit

### DISCUSSION THREAD

**Discussion by ricardogreis** [bluenviron/mediamtx#2319] [Questions]
Hello!
I'm working on a project to monitoring desktops. Using ffmpeg and mediamtx I'm close to getting it, but bandwidth from mediamtx and client (player) is too high ~1Mb/s per client using hlsVariant lowLatency. 

Here is my ffmpeg command on windows: 
ffmpeg -f dshow -i audio="YOUR_DEVICE" -c:a aac -filter_complex ddagrab=0,hwdownload,format=bgra,framerate=4 -c:v libx264 -pix_fmt yuv420p -crf 40 -bf 0 -y -f rtsp rtsp://mediamtx:8554/test

I'm using new ffmpeg version with Desktop Duplication API and how you can see, low quality and framerate. 
My monitor is 1920x1080. This command consumes a bandwidth of ~130Kb/s (ok to me). 

Could you help me to get the smallest bandwidth, with the closest time to live ?

**Comment by aler9** [comment]
Hello, if you want to control the bitrate explicitly and minimize latency, the README recommends using the ultrafrast preset and the `-b:v` flag, so:

```
ffmpeg -f dshow -i audio="YOUR_DEVICE" -c:a aac -filter_complex ddagrab=0,hwdownload,format=bgra,framerate=4 -c:v libx264 -pix_fmt yuv420p -preset ultrafast -b:v 600k -f rtsp rtsp://mediamtx:8554/test
```

[a comment by an automated account is not shown]
